russian-gost-r-705-2008-numeric-citations-alphabetical-bibliography.csl

```diff
@@ -0,0 +1,175 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" default-locale="ru-RU" et-al-min="4" et-al-use-first="1">
```

Comment:
by convention, we'd place the ```et-al-min="4" et-al-use-first="1"``` in the bibliography section. In other words, delete it here, then include it in line 109 below, so that that reads ```<bibliography et-al-min="4" et-al-use-first="1"/>```